spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe 'GET users#show', type: :feature do
```

Comment:
- Okay 👍 
![Capture](https://user-images.githubusercontent.com/63972670/215722284-eb457ba2-edd4-4279-a9ca-fd2a1041dd20.JPG)

Reply:
Thanks for the approval. 👍